one_fm/one_fm/custom/employee.json

```diff
@@ -674,7 +737,7 @@
    "name": "Employee-one_fm__highest_educational_qualification",
    "no_copy": 0,
    "non_negative": 0,
-   "options": "\nPost Graduate\nMasters\nGraduate\nDiploma\nUnder Graduate\nHigh School",
+   "options": "\nGraduate\nMasters\nPost Graduate\nUnder Graduate\nHigh School\nOther",
```

Comment:
Please check the field values in Employee, to make sure there is no changes other than the story demanded, like this.

Reply:
I have resolved this